### ISSUE TITLE
[Checkbox] unable to activate/check off a checkbox when keyboard tabbing in Safari

### ISSUE THREAD

**Issue by mollieru** [18F/web-design-standards#562]
*Steps to recreate*
1. Use keyboard tabbing to navigate the 'Checkboxes'
2. Try hitting 'enter' to activate a 'check' 

http://screencast-o-matic.com/watch/coQo3CfdAn
*Expected behavior*
Tabbing throughout the 'Checkboxes' - 'enter' should activate a check box

*Tested on*
OS 10.8
Safari 6.2

**Comment by angel** [comment]
Also doesn't work for Chrome.

@mollieru @carodew @colinpmacarthur Not sure if these should be in the same issue, but since they all fall under form controls in the bug testing doc: 
I can only tab to the selected radio button. Not sure if that's the right behavior we want. Also, enter does nothing for the dropdown too. Should you be able to open and select there with just the enter key?

**Comment by carodew** [comment]
Expected behavior I think is to space bar to select/deselect.
